python/rules_engine.py

```diff
@@ -190,7 +190,8 @@ def s302(self):
 		"""The reported Calendar Year does not match the filing year indicated at the start of the filing."""
 		field = "calendar_year"
 		edit_name = "s302"
-		fail_df = self.ts_df[self.ts_df.calendar_year != str(self.year)]
+		year = str(self.year)
```

Comment:
Does this actually change how it functions? The logic here looks the same, curious if it's hiding something else?

Reply:
It has the same logic but the change converts all variables in the check to a string, so the validation uses all similarly-typed variables.